docs/api/webview-tag.md

```diff
@@ -877,6 +894,7 @@ webview.addEventListener('close', () => {
 
 Returns:
 
+* `frameId` [number, number] - pair of `[processId, frameId]`.
 * `channel` String
 * `args` any[]
```

Comment:
Since this is &lt;webview>, this isn't actually a new positional argument, right? It's a new property on the `event` object?

Reply:
yes, the reason why I kept the array is to allow `webview.sendToFrame(event.frameId, ...`, otherwise you would have to do `webview.sendToFrame([event.frameProcessId, event.frameRoutingId], ...`